device/arista/x86_64-arista_7800_sup/Arista-7804R3-FM/0/ramon-a7800-7804r3-fm.config.bcm

```diff
@@ -0,0 +1,223 @@
+appl_param_module_id=300
+
+dpp_db_path=/usr/share/bcm/db
+port_init_cl72=1
+serdes_lane_config_cl72_auto_polarity_en=1
+serdes_lane_config_cl72_restart_timeout_en=0
+port_init_speed=53125
+
+appl_enable_intr_init=0
+bcm_linkscan_interval=0
+bcm_stat_interval=0
```

Comment:
One of the suggestions from Broadcom during the DNX bring up was to set these two 
bcm_stat_interval = 1000000
polled_irq_mode=0

I see irq_mode is set to 0 in the bcm files, but bcm_stat_interval  is set to 0, Was there any high CPU usage seen on the DUT due to stats polling ?

Reply:
I believe we disabled this setting to ease the bringup process and did not consider re-enabling it since.
I just tried enabling this setting and it seems to be fine. There's of course a higher CPU usage but it's reasonable.
I'll update the PR with the change you suggested.